[2024-06-15T20:59:25.852+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-15T20:59:25.888+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ran_info_api.format_response manual__2024-06-15T20:57:41.880945+00:00 [queued]>
[2024-06-15T20:59:25.903+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ran_info_api.format_response manual__2024-06-15T20:57:41.880945+00:00 [queued]>
[2024-06-15T20:59:25.904+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-06-15T20:59:26.119+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): format_response> on 2024-06-15 20:57:41.880945+00:00
[2024-06-15T20:59:26.133+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=7280) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-15T20:59:26.136+0000] {standard_task_runner.py:63} INFO - Started process 7290 to run task
[2024-06-15T20:59:26.132+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ran_info_api', 'format_response', 'manual__2024-06-15T20:57:41.880945+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/rand_request_api.py', '--cfg-path', '/tmp/tmp72k3ohm2']
[2024-06-15T20:59:26.137+0000] {standard_task_runner.py:91} INFO - Job 63: Subtask format_response
[2024-06-15T20:59:26.272+0000] {task_command.py:426} INFO - Running <TaskInstance: ran_info_api.format_response manual__2024-06-15T20:57:41.880945+00:00 [running]> on host d4d3ae2e009b
[2024-06-15T20:59:26.482+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='protectorate' AIRFLOW_CTX_DAG_ID='ran_info_api' AIRFLOW_CTX_TASK_ID='format_response' AIRFLOW_CTX_EXECUTION_DATE='2024-06-15T20:57:41.880945+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-15T20:57:41.880945+00:00'
[2024-06-15T20:59:26.484+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-15T20:59:26.508+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-15T20:59:26.509+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/rand_request_api.py", line 17, in format_response
    result = res['results'][0]
             ~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
[2024-06-15T20:59:26.525+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=ran_info_api, task_id=format_response, run_id=manual__2024-06-15T20:57:41.880945+00:00, execution_date=20240615T205741, start_date=20240615T205925, end_date=20240615T205926
[2024-06-15T20:59:26.576+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 63 for task format_response (string indices must be integers, not 'str'; 7290)
[2024-06-15T20:59:26.592+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-15T20:59:26.630+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-15T20:59:26.735+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
